plugins/scenario/hu.bme.mit.gamma.scenario.statechart.generator/src/hu/bme/mit/gamma/scenario/statechart/generator/AbstractContractStatechartGeneration.xtend

```diff
@@ -74,6 +75,7 @@ abstract class AbstractContractStatechartGeneration {
 	protected val extension ExpressionUtil exprUtil = ExpressionUtil.INSTANCE
 	protected val extension ScenarioStatechartUtil scenarioStatechartUtil = ScenarioStatechartUtil.INSTANCE
 	protected val StatechartUtil statechartUtil = StatechartUtil.INSTANCE
+	protected val ActionUtil actionUtil = ActionUtil.INSTANCE
```

Comment:
`ActionUtil` is the superclass of `StatechartUtil`, so the introduction of this attribute is unnecessary.